src/contributing/void-docs/submitting.md

```diff
@@ -0,0 +1,56 @@
+## Submitting Changes
+
+### Requirements
+
+To clone the repository and push changes git is required.
+
+```
```

Comment:
We're really trying to avoid having these package install commands all over the place. I think [git(1)](https://man.voidlinux.org/git.1) in this paragraph would be good enough. Also I think this could just be worked into the `Forking` section at that point.